### PR TITLE
keploy 0.7.6 (new formula)

### PR DESCRIPTION
**Pull request by Pradhyuman-sharma**
Updated the formula to the required changes needed last time.

### CODE REVIEW

**Comment by chenrui333** [comment]
do rebase instead of merge to update your PR.

**Comment by Pradhyuman-sharma** [comment]
done

**Comment by Pradhyuman-sharma** [comment]
Any update on this?

**Comment by chenrui333** [comment]
can you please do the rebase to the latest master branch rather than doing the merge update?

**Review by chenrui333 — APPROVED** [review]

File: Formula/keploy.rb [hunk 1]
```diff
@@ -23,15 +23,24 @@ def install
...
     system "go", "build", *std_go_args, "./cmd/server"
   end
+
   test do
+    require "pty"
+
     port = free_port
-    pid = fork do
-      assert_match("👍 connect to http://localhost:#{port}/ for GraphQL playground",
-                   shell_output("export PORT=#{port} && #{bin}/keploy"))
```
Comment (chenrui333) [inline Formula/keploy.rb]: This test is actually false positive as I did not see any `👍 connect to http://localhost:#{port}/ for GraphQL playground` in the server stdout

**Comment by chenrui333** [comment]
@Pradhyuman-sharma, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**